warehouse/query-core/src/main/java/datawave/query/jexl/visitors/whindex/SplitGeoWaveFunctionVisitor.java

```diff
@@ -37,7 +38,12 @@ public static <T extends JexlNode> T apply(T script, MetadataHelper metadataHelp
     public Object visit(ASTFunctionNode node, Object data) {
         JexlArgumentDescriptor descriptor = JexlFunctionArgumentDescriptorFactory.F.getArgumentDescriptor(node);
         if (descriptor instanceof GeoWaveFunctionsDescriptor.GeoWaveJexlArgumentDescriptor) {
-            Set<String> fields = descriptor.fields(metadataHelper, Collections.emptySet());
+            Set<String> fields = null;
+            try {
+                fields = descriptor.fields(metadataHelper, Collections.emptySet());
+            } catch (TableNotFoundException | InstantiationException | IllegalAccessException e) {
+                throw new RuntimeException(e);
```

Comment:
i am really thinking we should handle these exceptions in the method...

Reply:
Handled in latest update